extension/options/history/history.test.js

```diff
@@ -0,0 +1,93 @@
+/* globals test, expect */
+
+require("fake-indexeddb/auto");
+
+import { Database } from "../../history.js";
+
+global.log = (function() {
```

Comment:
I'm wondering if the global log export isn't available under the test scopes...
Regardless, this looks good

Reply:
Yea, unfortunately it seemed that the original global `log` was not available in the test scope which would cause some errors since `history.js` makes calls using `log`. I felt that defining it myself would be the best option rather than trying to rework how `log` is used in the entire repo.

Reply:
We can wait for Ian's reply, and see how he feels about this. For the meantime, looks great!